cattle/utils.py

```diff
@@ -110,10 +117,16 @@ def _marshall_to_python(self, data):
             return json.loads(data)
 
     def _get(self, url):
-        with closing(urllib2.urlopen(url, timeout=5)) as resp:
-            if resp.code == 200:
-                data = resp.read()
-                return self._marshall_to_python(data)
+        try:
+            with closing(urllib2.urlopen(url, timeout=5)) as resp:
+                if resp.code == 200:
+                    data = resp.read()
+                    return self._marshall_to_python(data)
+        except:
+            log.exception(
+                "Could not get stats from cAdvisor at: {0}".format(url))
+
```

Comment:
Bill, can you double check that all methods that call get_containers, have not None check?